ksmt-core/src/main/kotlin/org/ksmt/expr/rewrite/simplify/KExprSimplifier.kt

```diff
@@ -301,6 +318,9 @@ inline fun <T : KSort, A0 : KSort, A1 : KSort> KExprSimplifierBase.simplifyExpr(
     { transformExprAfterTransformed(expr, a0, a1) { ta0, ta1 -> ctx.simplifier(ta0, ta1) } }
 )
 
+/**
+ * Specialized version of [simplifyExpr] for expressions with 3 arguments.
```

Comment:
In some comments here you use `3`, and in some `three`. I think the last one is better (and "fourth" is better than `4`). Can you change it, please?